pkg/httperrors/httperrors_test.go

```diff
@@ -41,3 +43,57 @@ func TestHTTPError(t *testing.T) {
 		})
 	}
 }
+
+func TestGetHTTPResponseFromErr(t *testing.T) {
```

Comment:
The function name is quite verbose. It leads to a long function name:

```go
return httperrors.GetHTTPResponseFromErr(c, err)
```

Any way this could be slimmed down so it looks like:

```go
return httperrors.GetError(c, err)
```

Reply:
In favor of longer method names as more descriptive

Reply:
happy with long function names. As projects grow the more descriptive the method and function the better

Reply:
I am also in favour of longer descriptive function names.